terraspace/app/stacks/ecs-cluster/network.tf

```diff
@@ -0,0 +1,41 @@
+data "aws_availability_zones" "available" {
+}
+
+module "vpc" {
+  source               = "terraform-aws-modules/vpc/aws"
+  version              = "~> 3.0"
```

Comment:
Ideally this would be pinning, but this tool's infra isn't meant to be used forever

Reply:
Whenever this stack is destroyed, so will the VPC.
I'm not sure if coupling that with Pickup infra would be ideal.